terraform/ec2/main.tf

```diff
@@ -405,9 +405,10 @@ resource "null_resource" "setup_sample_app_and_mock_server" {
   provisioner "remote-exec" {
     inline = [
       "sudo yum update -y",
-      "sudo amazon-linux-extras install docker -y",
+      "sudo yum install -y docker",
       "sudo service docker start",
       "sudo usermod -a -G docker ec2-user",
+      "sudo ln -s /usr/lib/libcrypt.so /usr/lib/libcrypt.so.1",
```

Comment:
Should we [install](https://github.com/lando/lando/issues/3326#issuecomment-1625339749) instead of symbolic link?

Reply:
I marked it as a second solution in case any issues come up for symbolic linking. Was not sure if `yum` was included with `AL2023` so went with this.

Reply:
Update: The referred solution is better so changed it.